packages/story-editor/src/utils/isOffCanvas.ts

```diff
@@ -21,8 +21,9 @@ import {
   PAGE_HEIGHT,
   PAGE_WIDTH,
 } from '@googleforcreators/units';
+import type { Element } from '@googleforcreators/elements';
 
-export function isOffCanvas(selectedElement) {
+export function isOffCanvas(selectedElement: Element) {
```

Comment:
If only x, y, width and height are used, this can be typed more loosely as `ElementBox` from the unit package.

Reply:
Fixed.